spec/FrameSpec.js

```diff
@@ -0,0 +1,87 @@
+describe ('Frame', function () {
+
+  var frame;
+
+  beforeEach(function () {
+
+    frame = new Frame
+
+    });
+
+  describe ('Scoring', function () {
+    it ('returns 0 for two gutter balls', function() {
+      frame.enterPins(0);
+      frame.enterPins(0);
+      expect(frame.getScore()).toEqual(0);
+    });
+
+    it ('returns score for two scoring rolls', function() {
```

Comment:
Clear progression from the simplest possible version, very readable, gives an overview of your development process